wordpress-mobile/WordPress-iOS
# New Reader Detail: CSS fixes

Review only after #14339 is merged.

This PR:

- Center images in small devices so they don't appear in the left if they're small
- Adds the color for the captions
- Adds a structure to refresh the CSS every two days if the user is online (won't affect offline)
- Only display images after they have been fully loaded, to simulate a native feel
- Do not display images broken, again, to have a native feel

To test

* Check this post, the image should be center-aligned: https://made4testing0318.blog/2020/04/17/another-image-block/
* In the same post, you can check the caption color
* Open a post full of images and check that they appear as a whole instead of slowly appearing
* Open a post with a broken image and check that the image doesn't appear, space it's there, but no broken images icons are displayed

PR submission checklist:
